legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/planner/BulkLoadPlanner.java

```diff
@@ -154,4 +162,58 @@ private Selection getRowsBasedOnAppendTimestamp(Dataset dataset, String field, S
         FunctionImpl countFunction = FunctionImpl.builder().functionName(FunctionName.COUNT).addValue(All.INSTANCE).alias(alias).build();
         return Selection.builder().source(dataset.datasetReference()).condition(condition).addFields(countFunction).build();
     }
+
+    public static String jsonifyBatchSourceInfo(List<String> files)
```

Comment:
Would LogicalPlanUtils be a better place to hold this?

Reply:
Made it private non static